### PR TITLE
Update front-end and back-end 

### PR DESCRIPTION
**Pull request by ryanminjaeshin**
1. Update schema to camel casing. 
2. Update the App component with GET request working - fetching a product info details. 
3. Update the ProductDetails component.
4. Update the ProductImages component.
5. Add a proxy in the webpack.

Please ignore files in top_bar_components. This will be my stretch goal. Also, don't mind testing files.

Thanks!

@jonfu86 @karebae @rickboot

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,53 +1,43 @@
...
+  componentDidMount() {
+    this.getOneProductDetails(1);
   }
 
-  subtractCounter() {
-    this.setState((previousState) => ({
-      counter: previousState.counter - 1,
-    }));
+  getOneProductDetails(id) {
+    axios.get(`http://localhost:8080/products/${id}`)
```
Comment (karebae) [inline client/src/App.jsx:22]: YAYY big step!!! also Axios!! we love axios

File: client/src/App.jsx [hunk 2]
```diff
@@ -1,53 +1,43 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-// import Counter from './Counter';
+import axios from 'axios';
 
-class App extends React.Component {
+// import TopNavBar from './components/top_bar_components/TopNavBar/TopNavBar';
+import ProductDetails from './components/product_details_components/productDetails/ProductDetails';
```
Comment (karebae) [inline client/src/App.jsx:5]: folder naming conventions are totally individual preference, but i've personally been following Airbnb's naming conventions for folders as well as components. https://github.com/airbnb/javascript/tree/master/react#naming. TL;DR: they use camelcase for everything. Again, personal preference though!! Whatever works for you

File: client/src/components/top_bar_components/DropdownMenus/logo.svg [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (karebae) [inline client/src/components/top_bar_components/DropdownMenus/logo.svg:1]: i have to learn how to do this! thanks for reminding me

File: database/index.js [hunk 4]
```diff
@@ -16,45 +16,46 @@ db.once('open', () => {
...
   discount: Number,
   review: {
-    number_of_reviews: Number,
-    five_star_reviews: Number,
-    four_star_reviews: Number,
-    three_star_reviews: Number,
-    two_star_reviews: Number,
-    one_star_reviews: Number,
-    average_rating: Number,
+    numberOfReviews: Number,
```
Comment (karebae) [inline database/index.js:24]: i had to do this too LOL

**Review by karebae — APPROVED** [review]
Looks good Ryan! I know you're feeling behind but I can see that you're working really hard and I don't think you're giving yourself enough credit for the work you're getting done day-to-day. I saw you were up at like 10PM yesterday (also, I know you approved my pull request two days ago at like 2:00AM!!!) trying to get some things to work - that's a ton of dedication you're putting in and you should be really proud of that. Your hard work will pay off. Keep pushing!!

File: client/src/App.jsx (hunk 2 above)
Comment (ryanminjaeshin) [reply client/src/App.jsx:5]: @karebae Thanks Karin!! I will look into airbnb style guide. I totally missed that they have naming conventions for folders. 👍

File: client/src/App.jsx (hunk 1 above)
Comment (ryanminjaeshin) [reply client/src/App.jsx:22]: @karebae Yeah. This is first time for me :) I will love it too.

**Pull request merged**